### PR TITLE
docs: Add Lua-only key mappings examples

### PR DESCRIPTION
**Pull request by sestrella**
# Description

Adding some Lua-only key mapping examples taking advantage of NeoVim 0.7 features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +132,16 @@ nnoremap <leader>fb <cmd>lua require('telescope.builtin').buffers()<cr>
 nnoremap <leader>fh <cmd>lua require('telescope.builtin').help_tags()<cr>
 ```
 
+Using Lua:
+
+```lua
+local builtin = require('telescope.builtin')
+vim.keymap.set('n', 'ff', builtin.find_files, {})
```
Comment (Conni2461) [inline README.md:139]: mappings dont have parity with the viml because of missing `noremap = true`

Comment (sestrella) [reply README.md:139]: @Conni2461 thank you for the comment, however, I don't think `noremap` is a valid option in the context of `vim.keymap.set`. There is an option called `remap` with `false` as the default value which I think is equivalent to `noremap = true` in the context of `nvim_set_keymap`


image

Comment (Conni2461) [reply README.md:139]: image

It clearly links to `map-arguments`, noremap should be a map argument.  But remap clearly meantions that the default is already false. maybe the neovim team changed the default to the better.

**Comment by Conni2461** [comment]
Thanks :)

CI failure is unrelated and already fixed on master

**Pull request merged**